www/janf04/semestralka1/hraci.html

```diff
@@ -27,159 +27,161 @@ <h1>Galacticos FC</h1>
         </nav>
     </header>
 
-    <div class="wrapper">
-        <h2>Brankáři</h2>
-        <div class="container">
-            <div class="gkeepers">
-                <div class="float-box">
-                    <h3>Marek Marušák</h3>
-                    <div class="player-img">
-                        <img src="pics/mara.jpg" alt="mara">
-                    </div>
-                    <div class="player-info">
-                        <p>Výška: 178 cm</p>
-                        <p>Číslo dresu: 1</p>
-                        <p>Přezdívka: Mára</p>
-                    </div>
+    <main>
```

Comment:
myslel jsem `<main class="wrapper">`

Reply:
já mám ale na stránce hraci.html víc wrapperů, takže by tam bylo víc mainů, což je špatně

Reply:
aha, hm, uz me to prestalo bavit tak asi dobry

Reply:
okej :D